src/basic_recipes/contours.jl

```diff
@@ -264,6 +264,7 @@ function plot!(plot::T) where T <: Union{Contour, Contour3d}
         transform_marker = false
     )
 
+    transfunc = transform_func(plot)
     lift(plot, scene.camera.projectionview, transformationmatrix(plot), scene.viewport,
```

Comment:
Better to lift on transform_func_obs(plot), since the transform func can change arbitrarily during display.